tests/PHPUnit/Integration/ServeStaticFileTest.php

```diff
@@ -336,6 +336,7 @@ public function test_compressionCache()
         curl_setopt($curlHandle, CURLOPT_RETURNTRANSFER, true);
         curl_exec($curlHandle);
         curl_close($curlHandle);
+        sleep(1);
```

Comment:
Do you have a good reason for slowing down the tests by waiting a second here?

Reply:
@sgiehl was failed, but figure it out, it was caused by else, revert it